scripts/deploy.js

```diff
@@ -16,6 +16,15 @@ const spawnWithErrorHandling = (...args) => {
   }
 };
 
+let tagname = 'latest';
+if (version.includes('alpha')) {
```

Comment:
Maybe instead of those cases it's better to use regex for that: `-(.+)\.` so it will match for instance `beta` from `1.13.0-beta.3`, `rc` from `1.0.0-rc.0` and so on.